templates/ai-ml.html

```diff
@@ -34,18 +34,30 @@ <h2>UbuntuでAIモデルを開発する</h2>
 
 <section class="p-strip--light is-bordered">
   <div class="u-fixed-width">
-    <div class="p-logo-section">
+    <div class="p-logo-section has-misaligned-images">
       <p class="p-logo-section__title">
         人工知能分野のリーダーはUbuntuを選択しています
       </p>
       <div class="p-logo-section__items">
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/bac02008-Amazon.com-Logo.svg" alt="Amazon"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/b0716ed5-Google_logo.svg" alt="Google"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/07c6d390-microsoft_azure_logo.svg" alt="Microsoft"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/d3ee88a5-2018-logo-tesla.svg" alt="Tesla"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/c346c103-openai.png" alt="OpenAI"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/9eeeebca-logo-ibm.svg" alt="IBM"></div>
-        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/79974738-facebook-1.svg" alt="Facebook"></div>
+        <div class="p-logo-section__item"><img class="p-logo-section__logo" src="https://assets.ubuntu.com/v1/0adc7049-jupyter-logo.png" alt="Jupyter"></div>
```

Comment:
Can you not make it look similar to how it looks at https://ubuntu.com/ai? Right now the logos are too small.

Reply:
Bumping to latest vanilla seems to have done the trick. Fixed a few other broken logo sections as well